tests/test_sync_job_runner.py

```diff
@@ -58,6 +60,8 @@ def create_runner(
     source_changed=True,
     source_available=True,
     validate_config_exception=None,
+    sync_cursor=SYNC_CURSOR,
+    job_type=JobType.FULL,
```

Comment:
These two don't go together, right? Shouldn't FULL jobs have a None cursor?

Reply:
They are default values. It means, when the test runs, by default, the sync job is a full sync, and the data provider will return the sync cursor `SYNC_CURSOR` at the end of the sync (sync cursor needs to be updated at the end of both full and incremental sync).